histomicstk/cli/NucleiClassification/NucleiClassification.py

```diff
@@ -127,7 +127,7 @@ def main(args):
 
     with open(args.inputNucleiAnnotationFile) as f:
 
-        nuclei_annot_list = json.load(f)['elements']
+        nuclei_annot_list = json.load(f)['annotation']['elements']
```

Comment:
Elsewhere (in large_image_annotations), we try to be flexible about whether the annotation is wrapped in an outer record or not (e.g. `annot = json.load(f); nuclei_annot_list = annot.get('elements', annot.get('annotation', {}).get('elements'))`